tests/main/searching/task.yaml

```diff
@@ -34,8 +34,15 @@ execute: |
     .*"
     snap find test-snapd- | grep -Pzq "$expected"
 
+    echo "List of snaps in a section works"
+    # NOTE: this probably shows featured snaps, do not
+    # make any assumptions about the contents
+    test $(snap find --section=database | wc -l) -gt 1
```

Comment:
Nice. Hopefully we have at least 1 result for this section on arm ;)

Reply:
Hope so. According to https://uappexplorer.com/snap/ubuntu/postgresql10 lists `postgresql*` is listed for `arm64, armhf, amd64, i386`. OTOH querying the API directly at https://api.snapcraft.io/api/v1/snaps/search?&q=postgresql10 I see:
```
"architecture": [
     "amd64"
],
```

Reply:
they're separate revisions for each arch:
```
~$ for arch in arm64 armhf amd64 i386; do http https://api.snapcraft.io/api/v1/snaps/details/postgresql10 Accept:application/hal+json X-Ubuntu-Architecture:$arch X-Ubuntu-Series:16 fields==architecture,revision ; done
HTTP/1.1 200 OK
Content-Length: 71
Content-Type: application/json
Date: Thu, 14 Dec 2017 09:28:04 GMT
Server: gunicorn/19.7.1
X-Request-Id: ed7023db-45d7-487c-b1b0-6d1807206b66
X-VCS-Revision: 94e6de3

{
    "architecture": [
        "arm64"
    ],
    "package_name": "postgresql10",
    "revision": 35
}

HTTP/1.1 200 OK
Content-Length: 71
Content-Type: application/json
Date: Thu, 14 Dec 2017 09:28:05 GMT
Server: gunicorn/19.7.1
X-Request-Id: 813640f7-2ad1-440a-9ac4-322e5043e14c
X-VCS-Revision: 94e6de3

{
    "architecture": [
        "armhf"
    ],
    "package_name": "postgresql10",
    "revision": 36
}

HTTP/1.1 200 OK
Content-Length: 71
Content-Type: application/json
Date: Thu, 14 Dec 2017 09:28:05 GMT
Server: gunicorn/19.7.1
X-Request-Id: 5ec45d92-c660-4410-9f29-3e8e475ee1c9
X-VCS-Revision: 94e6de3

{
    "architecture": [
        "amd64"
    ],
    "package_name": "postgresql10",
    "revision": 34
}

HTTP/1.1 200 OK
Content-Length: 70
Content-Type: application/json
Date: Thu, 14 Dec 2017 09:28:06 GMT
Server: gunicorn/19.7.1
X-Request-Id: 7ede1281-78d3-4216-9e79-fe3b920c082a
X-VCS-Revision: 94e6de3

{
    "architecture": [
        "i386"
    ],
    "package_name": "postgresql10",
    "revision": 33
}

~$ 
```